monai/networks/utils.py

```diff
@@ -615,6 +626,104 @@ def convert_to_torchscript(
     return script_module
 
 
+def convert_to_trt(
+    model: nn.Module,
+    precision: str,
+    input_shape: Sequence[int],
+    dynamic_batchsize: Sequence[int] | None = None,
+    ir: str = "script",
+    filename_or_obj: Any | None = None,
+    verify: bool = False,
+    device: int | None = None,
+    rtol: float = 1e-2,
+    atol: float = 0.0,
+):
+    """
+    Utility to convert a model into a TensorRT engine based torchscript model with optional input / output data verification.
+
+    Args:
+        model: a source PyTorch model to convert.
+        precision: the weight precision of the converted TensorRT engine based torchscript models. Should be 'fp32' or 'fp16'.
+        input_shape: the input shape that is used to convert the model. Should be a list like [N, C, H, W] or
+            [N, C, H, W, D].
+        dynamic_batchsize: a list with three elements to define the batch size range of the input for the model to be converted.
+            Should be a list like [MIN_BATCH, OPT_BATCH, MAX_BATCH], default to None.
+        ir: the intermediate representation way to transform a pytorch module to a TensorRT engine based torchscript. Could
+            be chosen from `(script, trace)`, default to "script.
+        filename_or_obj: if not None, specify a file-like object (has to implement write and flush) or a string containing a
+            file path name to load the TensorRT engine based torchscript model for verifying.
+        verify: whether to verify the input and output of the TensorRT engine based torchscript model.
+        device: the target GPU index to convert and verify the model. If None, use #0 GPU.
+        rtol: the relative tolerance when comparing the outputs between the PyTorch model and TensorRT model.
+        atol: the absolute tolerance when comparing the outputs between the PyTorch model and TensorRT model.
+    """
+
+    if not has_torch_tensorrt:
+        raise ImportError("Cannot find the torch_tensorrt module, please install it before converting.")
+
+    if not torch.cuda.is_available():
+        raise Exception("Cannot find any GPU devices.")
+
+    if not input_shape:
+        raise ValueError("Missing the input shape for model convert.")
+
+    if not dynamic_batchsize:
```

Comment:
suggest to add a length check here, need to be 3, otherwise provide an informative error message